docs/modules/ROOT/pages/how-to/use-tilt.adoc

```diff
@@ -0,0 +1,201 @@
+= How to use Tilt with {product}
+
+image:logos/tilt.svg[role="related thumb right",alt="Tilt logo",width=120,height=120] https://tilt.dev/[Tilt] is a developer toolkit to enable teams to build microservices architectures faster on Kubernetes clusters.
+
+> The best indicator of a healthy development workflow is a short feedback loop.
+
+https://docs.tilt.dev/example_go.html[(Source: Tilt documentation)]
+
+This page will describe the required steps to use Tilt successfully with {product}.
+
+IMPORTANT: This information is provided as a courtesy for {product} users. We don't provide support for Tilt, and in case of issues, you are invited to contact the developers of Tilt at their https://github.com/tilt-dev/tilt[GitHub project].
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`tilt`:: The https://docs.tilt.dev/[Tilt documentation] explains how to install it in your operating system of choice.
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry.
+
+`docker`:: You can download it from https://www.docker.com.
+
+== Procedure
+
+This section explains the steps required to use Tilt with {product}.
+
+=== 1. Login to {product}
```

Comment:
That login procedure is possibly already and most certainly will be used on many places. Please move that to a dedicated "how do I log in" or make use of a partial (allow us to maintain it once but display it a many places).